mailpoet/lib/Subscribers/ImportExport/ImportExportRepository.php

```diff
@@ -118,7 +118,7 @@ public function insertMultiple(
       }, $columns);
 
       foreach ($item as $columnKey => $column) {
-        $parameters[$paramNames[$columnKey]] = $column;
+        $parameters[substr($paramNames[$columnKey], 1)] = $column;
```

Comment:
Are you removing the leading colon here in the loop? It would be great to add a comment about it.

Reply:
Good point. I added a comment [here](https://github.com/mailpoet/mailpoet/pull/5641/commits/2e8cb0a6e606b194cf708b7663afee6cf28fd4b7).